client/state/data-layer/wpcom-http/utils.js

```diff
@@ -83,7 +96,10 @@ const defaultOptions = {
  * @returns {?*} please ignore return values, they are undefined
  */
 export const dispatchRequest = ( initiator, onSuccess, onError, options ) => ( store, action ) => {
-	const { onProgress } = Object.assign( defaultOptions, options );
+	const {
+		fromApi,
+		onProgress,
+	} = Object.assign( defaultOptions, options );
```

Comment:
this is probably a bug. I don't think we want to be assigning into the `defaultOptions` do we?